kolibri/core/assets/src/vue/content-renderer/download-button.vue

```diff
@@ -1,12 +1,30 @@
 <template>
 
   <div class="dropdown">
-    <button>{{ $tr('downloadContent') }} &#9660</button>
-    <div class="dropdown-items">
-      <a class="dropdown-item" v-for="file in files" href="{{ file.download_url }}">
-        {{ file.preset + ' (' + prettifyFileSize(file.file_size) + ')' }}
-      </a>
-    </div>
+    <button
+      class="dropdown-button"
+      @click="toggleDropdown"
+      aria-haspopup="true">
+      {{ $tr('downloadContent') }} &#9660
+    </button>
+    <ul
+      class="dropdown-items"
+      :class="{ dropdownopen: dropdownopen }"
+      :aria-hidden="dropDownOpenText"
+      role="menu">
```

Comment:
Since the list is inside a button, it can be opened with both `Enter` and spacebar (so far so good), but in order to access the links with download options inside the list, one has to use the `TAB` key. It is, of course, expected to be able to access links with `TAB` key, no qualms with that either, but the problem is that this selection looks like an on-click **dropdown menu** to both sighted users who use keyboard navigation, and is announced as a menu to screen reader users (with `role="menu"`). And if something looks like a menu, and quacks like a menu, users will expect to navigate the options with arrow keys... :wink: